spirv-tools/tests/val.rs

```diff
@@ -0,0 +1,44 @@
+const SPIRV_BIN: &[u8] = include_bytes!("wgpu_example_shader.spv");
+
+fn validate_compiled(_input: &[u8]) -> Option<Result<(), spirv_tools::Error>> {
+    #[cfg(feature = "use-compiled-tools")]
```

Comment:
It might be worth adding `cfg_if` at some stage, lets you write this in a much nicer fashion.

```rust
cfg_if::cfg_if! {
    if #[cfg(feature = "use-compiled-tools")] {
       Some(..)
    } else {
       None
    }
}
```

Reply:
Yah, I was planning on fleshing out testing for this eventually, that would definitely be a part of the push.